views/networking/load_balancer/index.erb

```diff
@@ -0,0 +1,54 @@
+<% @page_title = "Load Balancers" %>
+<%== render("networking/tabbar") %>
+<% if @lbs.count > 0 %>
+  <div class="overflow-hidden rounded-lg shadow ring-1 ring-black ring-opacity-5 bg-white divide-y divide-gray-200">
+    <table class="min-w-full divide-y divide-gray-300">
+      <thead class="bg-gray-50">
+        <tr>
+          <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900 sm:pl-6">Name</th>
+          <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">Hostname</th>
+        </tr>
+      </thead>
+      <tbody class="divide-y divide-gray-200 bg-white">
+        <% @lbs.each do |lb| %>
+          <tr>
+            <td class="whitespace-nowrap py-4 pl-4 pr-3 text-sm font-medium text-gray-900 sm:pl-6" scope="row">
+              <a href="<%= @project_data[:path] %><%= lb[:path] %>" class="text-orange-600 hover:text-orange-700">
+                <%= lb[:name] %>
+              </a>
+            </td>
+            <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
+            <% if lb[:hostname] %>
+              <%== render("components/copieble_content", locals: { content: lb[:hostname], message: "Copied hostname" }) %>
+            <% else %>
+              Waiting for hostname to be ready
+            <% end %>
+            </td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  </div>
+  <div class="flex justify-end space-y-1 mt-6">
+    <% if has_project_permission("LoadBalancer:create")%>
+      <%== render(
+        "components/button", locals: { text: "Create Load Balancer", link: "load-balancer/create" })
+      %>
+    <% end %>
+  </div>
+<% else %>
+<div class="mt-6">
+  <%== render(
+    "components/empty_state",
+    locals: {
+      icon: "hero-circle-stack",
```

Comment:
not related icon for load balancers and private subnets

<img width="485" alt="Screenshot 2024-07-11 at 14 34 15" src="https://github.com/ubicloud/ubicloud/assets/993199/0dcedf47-d790-4e60-bfad-b9173df47c50">

<img width="488" alt="Screenshot 2024-07-11 at 14 34 18" src="https://github.com/ubicloud/ubicloud/assets/993199/8fe1aba1-6436-44cf-9e6f-2643491a61b8">